src/main/java/bio/terra/service/duos/SystemStatus.java

```diff
@@ -0,0 +1,8 @@
+package bio.terra.service.duos;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.Map;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Is this tag always needed for records?

Reply:
Good question, it isn't required but I like it in cases like this one.  I.E. we would really prefer a client provided by DUOS team, but because we're writing our own based off of the existing responses given by their API.  This avoids deserialization errors if they ultimately expand their response bodies, we'll just ignore any values we don't care about.

Reply:
(It's also nice if you know you only care about a small subset of existing fields!)

Reply:
Ah gotcha, thanks for explaining!